src/code-examples/tabs/tabs.html.js

```diff
@@ -0,0 +1,26 @@
+const html = `
+    <div class="tabs">
+      <nav role="tablist" aria-labelledby="tablist-1" class="manual">
+        <button id="tab-1" type="button" role="tab" aria-selected="true" aria-controls="tabpanel-1">Section 1</button>
+        <button id="tab-2" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-2" tabIndex="-1">Section 2</button>
+        <button id="tab-3" type="button" role="tab" aria-selected="false" aria-controls="tabpanel-3" tabIndex="-1">Section 3</button>
+      </nav>
+    </div>
```

Comment:
Nitpick ~ we can wrap all of the HTML in this div 😀

Reply:
Fixed here: https://github.com/sparkbox/accessible-components/pull/77/commits/daa797c6a07e0c0938a514bacbb27e062ebf25d3